source/slang/slang-ast-support-types.h

```diff
@@ -143,6 +143,9 @@ namespace Slang
         // Default case (usable for user-defined conversions)
         kConversionCost_Default = 500,
 
+	// Cost of converting an integer to int16_t
+        kConversionCost_IntegerToShort = 700,
```

Comment:
Maybe give it more general name IntegerTruncate?

Reply:
Yes I will do that.